Sources/XcodeProj/Scheme/XCScheme+Parallelizable.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+public enum Parallelizable: String {
```

Comment:
I'd declare this in an `XCScheme` extension and add some basic docs for what this `enum` represents/is supposed to be used for.